src/main/webapp/app/scores/exercise-scores.component.ts

```diff
@@ -207,7 +207,8 @@ export class ExerciseScoresComponent implements OnInit, OnDestroy {
     }
 
     goToBuildPlan(result: Result) {
-        this.sourceTreeService.goToBuildPlan(result.participation!);
+        // TODO: get the continuous integration URL as a client constant during the management info call
+        window.open('https://bamboobruegge.in.tum.de/browse/' + (result.participation! as ProgrammingExerciseStudentParticipation).buildPlanId);
```

Comment:
Is this the way to go? Especially if we are looking at an upcoming Jenkins integration, I think it is easier to build up this URL server-side.

Reply:
We have lots of null pointer exceptions on the server. As we have this hard coded in many places, it does not make sense to have a server call here that might even fail.

I think the TODO is the way to go. Load the BASE URL for the CI Service once and then build the URL on the client without additional server calls. However this is not important for now, we can fix this in a future PR